ui/src/components/tables/podStatusTable.tsx

```
@@ -0,0 +1,49 @@
+        <thead>
+          <tr class="bg-gray-800">
+            <th class="px-4 py-2 border-b text-left">Execution Order</th>
+            <th class="px-4 py-2 border-b text-left">Pod Name</th>
+            <th class="px-4 py-2 border-b text-left">Status</th>
+            <th class="px-4 py-2 border-b text-left">Exit Code</th>
+            <th class="px-4 py-2 border-b text-left">Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.details.pods.map((pod, i) => (
+            <tr class="border-b hover:bg-gray-500">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)